P5/Source/Guidelines/en/CMC-ComputerMediatedCommunication.xml

```diff
@@ -0,0 +1,1330 @@
+      taxonomy describes units that are performed (by a producer) and perceived (by a recipient) as
+      a continuous stream of behaviour. Units of this type can be performed as</p>
+    <list type="gloss">
+      <label rend="bold">spoken utterances,</label>
+      <item>i.e. stretches of speech which are produced to perform a speaker turn in a
+        conversation,</item>
+      <label rend="bold">bodily activity,</label>
+      <item> i.e. nonverbal behavior (gesture, gaze) produced to perform a speaker turn, either
+        performed by the real body of an interlocutor (e.g. in a video conference) or through the
+        virtual avatar of an interlocutor in a 3D environment,</item>
+      <label rend="bold">onscreen activities,</label>
+      <item> i.e. non-bodily forms of behaviour that are transmitted to the group of interacting or
+        coworking participants, for instance the editing of content in a shared text editor which
+        can be perceived by the other parties simultaneously (as may be the case in learning or
+        collaboration environments).</item>
+    </list>
+    <p>The right part of <ref target="#cmcunits-taxonomy">the taxonomy</ref> describes units in
+      which the production, transmission, and perception of contributions to CMC interactions are
+      organised in a strictly consecutive order: The content — verbal, nonverbal, or multimodal — of
+      the contribution has to be produced before it can be transmitted through a network and made
+      available on the computer monitor or mobile screen of any other party as a preserved and
+      persistent unit. We term this type of unit a <emph rend="italic">post</emph>. Posts occur in
+      two different variants: <list rend="bulleted">
+        <item>as <emph rend="bold">written or multimodal posts,</emph> which are produced with an
+          editor form that is designed for the composition of stretches of written text. Most
+          contemporary post-based CMC technologies provide features for the inclusion of graphic and
+          audiovisual content (emoji graphics, images, videos) into posts and even to produce posts
+          without verbal content (which then may consist only of emojis, an image, or a video file).
+          Written and multimodal posts are the standard formats for user contributions in primarily
+          text-based CMC genres and applications such as chat, SMS, WhatsApp, Instagram, Facebook, X
+          (Twitter), online forums, or Wikipedia talk pages.</item>
+        <item>as <emph rend="bold">audio posts</emph>, which are produced using a recording
+          function. In contrast to CMC units of the type <emph rend="italic">utterance</emph> which
+          are produced and transmitted simultaneously, audio posts first have to be recorded as a
+          whole and are then transmitted — as audio files — via the internet; the availability of
```

Comment:
```suggestion
          whole and are then transmitted—as audio files—through a network; the availability of
```

Reply:
Amended all em dashes for consistency in https://github.com/TEIC/TEI/pull/2537/commits/9407cd7f6a866ec8c7df720685569d12b39bbfc5